### ISSUE TITLE
Getting a weird Typescript error with Prisma plugin

### ISSUE THREAD

**Issue by LALauw** [hayes/pothos#717]
*edit fixed by changing strict to true in tsconfig
I'm currently setting up pothos with Nextjs and Prisma. Following a guide by Prisma themselves I get a weird Typescript error when defining the builder. I defined it as follows

````typescript
import SchemaBuilder from '@pothos/core';
import { DateResolver, DateTimeResolver } from 'graphql-scalars';
import PrismaPlugin from '@pothos/plugin-prisma';
import type PrismaTypes from '@pothos/plugin-prisma/generated';
import { prisma } from '../db/prisma';

export const builder = new SchemaBuilder<{
  Scalars: {
    Date: { Input: Date; Output: Date };
  };
  PrismaTypes: PrismaTypes;
}>({
  plugins: [PrismaPlugin],
  prisma: {
    client: prisma,
  },
});

builder.addScalarType('Date', DateTimeResolver, {});
builder.queryType({});
builder.mutationType({});
````

However I get red squigglies:
![image](https://user-images.githubusercontent.com/32265116/208997852-5d2ef9a4-676c-4c7e-9d78-b771acf2d224.png)
![image](https://user-images.githubusercontent.com/32265116/208997867-2362b87b-979d-4c14-921d-60d67097427d.png)

Not sure what the problem is here.

**Comment by LALauw** [comment]
Fixed it by going to tsconfig and changing strict to true. I do however have another problem when building the types. 
![image](https://user-images.githubusercontent.com/32265116/209000901-b801f9af-aad6-48bf-a31c-c8cbd2a0ef4c.png)

which is weird since I do have the type in my prisma schema
````typescript

model Person {
  id        String   @id @default(cuid())
  name      String
  createdAt DateTime @default(now())
}
````

**Comment by hayes** [comment]
Can you double check a few things:

* The Pothos generator was added to your Prisma schema
* You ran Prisma generate after adding the generator (and after the last time you installed new dependencies)
* You imported the generated Prisma types and added them to the builder
* If you open the file the Prisma types are being imported from, it actually contains the generated types

**Comment by LALauw** [comment]
> 

- The Pothos generator was added to your Prisma schema ✅
![image](https://user-images.githubusercontent.com/32265116/209005528-999d4008-f31f-431b-a9dc-cdb5a4a58a35.png)

- You ran Prisma generate after adding the generator (and after the last time you installed new dependencies) ✅
Just ran npx prisma generate again to be sure
![image](https://user-images.githubusercontent.com/32265116/209005477-8a62927b-beee-4e46-ae60-71d0b23a4365.png)

- You imported the generated Prisma types and added them to the builder ❗
Noticed the problem here, I used a custom output in the schema.prisma file: './pothos.ts' while I was still importing the generated.ts from the default route. So I changed the PrismaTypes import to the custom output and it works!

- If you open the file the Prisma types are being imported from, it actually contains the generated types ✅
The generated file in the output I wrote in the schema contains the generated types.

Thanks for the swift response, the double checking helped me 👍
